nixos/modules/installer/tools/nixos-generate-config.pl

```diff
@@ -514,6 +514,13 @@ sub in {
                 }
             }
         }
+        if ($dmUuid =~ /^LVM-/)
+        {
+            push @attrs, "  boot.initrd.services.lvm.enable = true;";
```

Comment:
It was mentioned on matrix that this might result in the line being added to the config multiple times if there are multiple LVM/md devices. If that's the case, we definitely need to guard against it.